tests/test_dmlrunner.py

```diff
@@ -19,6 +23,42 @@ def config_manager():
     )
     return config_manager
 
+
+@pytest.fixture
+def sample_transform_function():
+    return lambda x: x.drop_duplicates()
+
+@pytest.fixture
+def mnist_filepath():
+    return 'tests/artifacts/runner_scheduler/mnist'
+
+@pytest.fixture
+def small_filepath():
+    return 'tests/artifacts/runner_scheduler/test'
+
+def accurate_transform_test(dsm, expected_test_transformed, split):
+    """
+    Check that the data in this folder is the result of calling the transform function on
+    each csv in the raw data filepath
+    """
+    transform_dsm = dsm.get_transformed_data()
+    keys = list(transform_dsm.keys())
+    train = transform_dsm['train']
+    test = transform_dsm['test']
+    count = len(expected_test_transformed)
+    assert len(train) == int(split * count)
+    assert len(test) == int((1 - split) * count)
+    actual_test_transformed = train.append(test)
+    actual_test_transformed = actual_test_transformed.reset_index(drop=True)
+    merged_data = pd.merge(
+        actual_test_transformed,
+        expected_test_transformed,
+        on = ['a', 'b', 'c'],
+        how = 'inner'
+    )
+    assert len(actual_test_transformed) == count
+    assert len(merged_data) == count
```

Comment:
It seems like it is easier for this test to check element-wise, order-agnostic equality.

Reply:
Not using this function, will delete.